pkg/providers/aws/credentials.go

```diff
@@ -136,14 +137,24 @@ type Credentials struct {
 	PolicyName      string
 	AccessKeyID     string
 	SecretAccessKey string
+	RoleArn         string
+	TokenFilePath   string
 }
 
 //go:generate moq -out credentials_moq.go . CredentialManager
 type CredentialManager interface {
 	ReconcileProviderCredentials(ctx context.Context, ns string) (*Credentials, error)
-	ReconcileSESCredentials(ctx context.Context, name, ns string) (*Credentials, error)
-	ReoncileBucketOwnerCredentials(ctx context.Context, name, ns, bucket string) (*Credentials, *v1.CredentialsRequest, error)
-	ReconcileCredentials(ctx context.Context, name string, ns string, entries []v1.StatementEntry) (*v1.CredentialsRequest, *Credentials, error)
+	ReconcileBucketOwnerCredentials(ctx context.Context, name, ns, bucket string) (*Credentials, error)
```

Comment:
ReconcileBucketOwnerCredentials. Is this the one we put in the secret in the case of the managed-api?

Reply:
In case of managed-api this one is used by 3Scale, so we end up copying it over to a Secret in 3Scale ns.
I am not sure if this answers the question though.